ecommers/lib/ui/pages/product_page.dart

```diff
@@ -14,23 +15,30 @@ import 'package:ecommers/core/provider_models/index.dart';
 import 'package:ecommers/core/services/index.dart';
 
 class ProductPage extends StatefulWidget {
```

Comment:
why do u use stateful widget here?

Reply:
Because images for carousel control should be updated by clicking on color.